free/src/main/scala/cats/free/package.scala

```diff
@@ -4,4 +4,6 @@ package object free {
   /** Alias for the free monad over the `Function0` functor. */
   type Trampoline[A] = Free[Function0, A]
   object Trampoline extends TrampolineFunctions
+  type StacksafeKleisli[F[_], A, B] = FreeT[A => ?, F, B]
```

Comment:
The "S" in "Safe" is already capitalized in `StackSafeMonad`, so it should probably be capitalized here for consistency.